src/HTMLImports/path.js

```diff
@@ -11,27 +11,31 @@ HTMLImports.addModule(function(scope) {
 
 var CSS_URL_REGEXP = /(url\()([^)]*)(\))/g;
 var CSS_IMPORT_REGEXP = /(@import[\s]+(?!url\())([^;]*)(;)/g;
+var CSS_ABSOLUTE_URL_REGEXP = /^(http[s]?:)?\/\//i
```

Comment:
this won't work for schemes other than http(s), right? Is that a concern?
e.g. chrome extensions might use a chrome:// scheme